server/routerlicious/packages/local-server/src/localDeltaConnectionServer.ts

```diff
@@ -138,13 +137,6 @@ export class LocalDeltaConnectionServer implements ILocalDeltaConnectionServer {
         await this.ordererManager.close();
     }
 
-    /**
-     * Returns true if there are any received ops that are not yet ordered.
-     */
-    public async hasPendingWork(): Promise<boolean> {
```

Comment:
looks like a breaking change that will require a major version bump

Reply:
Restored in latest.